client/components/lists/listBody.js

```diff
@@ -60,6 +88,13 @@ BlazeComponent.extendComponent({
         type: 'cardType-card',
       });
 
+      // if the displayed card count is less than the total cards in the list,
+      // we need to increment the displayed card count to prevent the spinner
+      // to appear
+      const cardCount = this.data().cards(this.idOrNull(swimlaneId)).count();
+      if (pthis.cardlimit.get() < cardCount) {
```

Comment:
> In Lists and Swimlane mode, when I'm writing text to minicard, pressing Enter duplicates the same text to next card.
> 
> Pressing Enter should add card below, and go to card below, without duplicating text.

yep, typo: `pthis.cardlimit.get()` should be `this.cardlimit.get()`

This must have chimed in while I updated the comment above, sorry.